src/main/scala/com/typesafe/conductr/sbt/ConductRPlugin.scala

```diff
@@ -20,217 +22,67 @@ object ConductRPlugin extends AutoPlugin {
   import sbinary.DefaultProtocol.FileFormat
 
   val autoImport = Import
+  import ConductRKeys._
```

Comment:
In general, I've always found it better to be specific about which key I want to use within a plugin in order to avoid namespace clashes. Just a FYI, not terribly important.